mods/tuxemon/maps/taba_town.tmx

```diff
@@ -831,5 +865,20 @@
     <property name="cond40" value="is button_pressed K_RETURN"/>
    </properties>
   </object>
+  <object id="152" name="Son of a Witch" type="event" x="812.667" y="872" width="16" height="16">
```

Comment:
events and collisions should be aligned with the tile grid and should be whole numbers